src/Microsoft.TestPlatform.ObjectModel/RunSettings/RunConfiguration.cs

```diff
@@ -524,12 +572,19 @@ public static RunConfiguration FromXml(XmlReader reader)
                             break;
 
                         default:
-                            throw new SettingsException(
```

Comment:
Yes, it is a change in behavior. Now that TestHost is nuget package, we need to reduce coupling; this will allow a testrunner of newer version (with new runconfiguration settings) to run tests in older testhost.